pop-cli-for-appchains/guides/launch-a-chain/coretime.md

```diff
@@ -0,0 +1,51 @@
+---
+description: The following guide shows how to acquire coretime.
+---
+
+# Coretime
+
+In order to get a chain's block validated and finalised by the Relay chain it needs to acquire a [core](https://wiki.polkadot.network/docs/learn-agile-coretime).
+
```

Comment:
"pop allows users to obtain on demand coretime as easily as executing one command:"